modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -63,6 +63,38 @@ test('LPD-18485 Update account contact information fields', async ({
 	}
 });
 
+test('LPD-18484 Add account contact address', async ({
+	accountContactAddressPage,
+	accountsPage,
+	apiHelpers,
+	editAccountContactAddressPage,
+	editAccountPage,
+	page,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: 'test',
+		type: 'business',
+	});
+
+	await accountsPage.goto();
+
+	try {
+		await (await accountsPage.accountsTableRowLink(account.name)).click();
+		await editAccountPage.contactLink.click();
+		await accountContactAddressPage.addAddressesButton.click();
+		await editAccountContactAddressPage.updateAddress('address1', 'city');
+
+		await expect(
+			page.getByText('Success:Your request completed successfully.')
+		).toBeVisible();
+
+		await expect(page.getByText('address1city,')).toBeVisible();
```

Comment:
can you create a dynamic locator (so a function that accept the name) inside the page? And is it the `,` needed?